src/views/posts-index.js

```diff
@@ -0,0 +1,44 @@
+var RunView = Backbone.View.extend({
+  model: null,
+
+  tagName: 'li',
+
+  template(model) {
+    // Wire up model events
+    return `<li>${model.date} - ${model.time}</li>`;
+  },
+
+  render() {
+    // Sets up the DOM
+    this.$el.html(
+      `${this.model.get('date')} - ${this.model.get('time')}
+
+        <a href="#new"><i class="fa fa-plus-circle"></i></i></a>`);
```

Comment:
Move this out of your item view and into the collection view below. So you can append `<li><a href="#new"><i class="fa fa-plus-circle"></i></a></li>` to the collection view.